.semaphore/semaphore.yml

```diff
@@ -16,4 +16,5 @@ blocks:
             - 'export PATH=/home/semaphore/go/bin:$PATH'

```

Comment:
use `export "GOPATH=$(go env GOPATH)"` instead of hardcode the GOPATH for line 15

Reply:
include `GOPATH` in your `PATH` too

Reply:
Also please use valid name for line 2 `name` instead of `Go`. `telemetry-reporter-go` would be fine